deflate.c

```
@@ -565,6 +565,15 @@ int32_t Z_EXPORT PREFIX(deflatePending)(PREFIX3(stream) *strm, uint32_t *pending
     return Z_OK;
 }
 
+/* ========================================================================= */
+int32_t Z_EXPORT PREFIX(deflateUsed)(PREFIX3(stream) *strm, int32_t *bits) {
```

Cppcheck (2.21.0)

[style] The function 'zng_deflateUsed' is never used.

(unusedFunction)